test/markdownlint-cli2-test-cases.js

```diff
@@ -444,6 +444,18 @@ const testCases =
     "usesRequire": true
   });
 
+  testCase({
+    "name": "markdownlint-cli2-config-option-with-extends",
+    "args": [
+      "--config",
+      "configs/custom.markdownlint-cli2.jsonc",
+      "viewme.md"
+    ],
+    "exitCode": 0,
```

Comment:
Good. To be sure, deleting the `extends` should produce lint issues that fail this test, right?

Reply:
Yes, it does fail.